Upstream supports ncursesw but doesn't properly detect it. Let's do things the
right way and allow optional clients to be built.

https://gitlab.com/gpsd/gpsd/-/merge_requests/295

diff --git a/SConscript b/SConscript
index 345ead71550069f8fe976fab3f529f56b9e59e2b..e9c22358e0834404b1f1cfbbb1a0fca3ff58b658 100644
--- a/SConscript
+++ b/SConscript
@@ -1023,6 +1023,10 @@ if not cleaning and not helping:
             ncurseslibs = pkg_config('ncurses', rpath_hack=True)
             if config.CheckPKG('tinfo'):
                 ncurseslibs += pkg_config('tinfo', rpath_hack=True)
+        elif config.CheckPKG('ncursesw'):
+            ncurseslibs = pkg_config('ncursesw', rpath_hack=True)
+            if config.CheckPKG('tinfo'):
+                ncurseslibs += pkg_config('tinfo', rpath_hack=True)
         # It's not yet known whether rpath_hack is appropriate for
         # ncurses5-config.
         elif WhereIs('ncurses5-config'):
